graylog2-web-interface/src/components/sidecars/hooks/useBasicSidecarUser.ts

```diff
@@ -0,0 +1,44 @@
+import { useQuery } from '@tanstack/react-query';
```

Comment:
[INFO] Checking licenses...
Warning:  Missing header in: /home/runner/work/graylog2-server/graylog2-server/graylog2-web-interface/src/components/sidecars/hooks/useBasicSidecarUser.ts